mod/wikirate/set/type/file.rb

```diff
@@ -4,7 +4,9 @@
     if args[:structure]
       process_content _render_raw(args)
     else
-      super args
+      handle_source args do |source|
+        card_url source
+      end
```

Comment:
I tried to call `super args` to show the default core view when there is no structure. I found that the `view :core` in `mod/wikirate/set/type/file.rb` will rewrite the `view :core` in `file.rb` in wagn and the `super args` will call the upper layer's core view from `format do` instead of `format :html do` in file.rb . I found a new way to overcome this problem by having one more submodule. `super` is back